frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelCorrelationTable.tsx

```diff
@@ -300,17 +314,25 @@ export function FunnelCorrelationTable(): JSX.Element | null {
 const CorrelationActionsCell = ({ record }: { record: FunnelCorrelation }): JSX.Element => {
     const { insightProps } = useValues(insightLogic)
     const logic = funnelLogic(insightProps)
-    const { excludeEventProperty } = useActions(logic)
-    const { isEventPropertyExcluded } = useValues(logic)
-    const eventName = record.event.event.split('::')[0]
-    const propertyName = (record.event.event || '').split('::')[1]
+    const { excludeEventProperty, excludeEvent } = useActions(logic)
+    const { isEventPropertyExcluded, isEventExcluded } = useValues(logic)
+    const components = record.event.event.split('::')
```

Comment:
non-blocking

should we be using `parseEventAndProperty` here?

Reply:
Hmm, the property format we get out of it isn't what we want here :$